Makefile

```diff
@@ -13,7 +13,7 @@
 #   make check
 #
 # to compile and install Red Alert, Tiberian Dawn, and Dune 2000, run:
-#   make [prefix=/foo] [bindir=/bar/bin] install
+#   make [prefix=/foo] [bindir=/bar/bin] [TARGETPLATFORM=unix-generic] install
```

Comment:
To be consistent with the compile comment above we should split `TARGETPLATFORM=unix-generic` to another comment.